translator/translate/otel/processor/awsappsignals/translator.go

```diff
@@ -82,16 +80,17 @@ func (t *translator) Translate(conf *confmap.Conf) (component.Config, error) {
 		}
 	} else {
 		// Non-kubernetes environment detected.
-		ctx := context.CurrentContext()
-		if ctx.Mode() == config.ModeEC2 {
-			cfg.Resolvers = []appsignalsconfig.Resolver{
-				appsignalsconfig.NewEC2Resolver(hostedIn),
-			}
-		} else {
-			cfg.Resolvers = []appsignalsconfig.Resolver{
-				appsignalsconfig.NewGenericResolver(hostedIn),
-			}
-		}
+		//ctx := context.CurrentContext()
```

Comment:
Should we change the EC2 unit test case in awsappsignals? I thought it would fail in the EC2 mode.